libraries/conf_helpers.rb

```diff
@@ -46,7 +46,8 @@ def self.parse_config(conf_body)
         case ln
         when /^\s*\[([^#]+)\]\s*/
           current_section = Regexp.last_match[1]
-        when /^\s*([^#]+?)\s*=\s*([^#]+?)\s*(?:#.*)?$/
+          current_config[current_section] ||= {}
+        when /^\s*([^#]+?)\s*=\s*([^#\v]*?)\s*(?:#.*)?$/
```

Comment:
Ah wait... it's not the vertical tab addition it's the change of `[^#]+?` to `[^#]*?` which is doing the magic here. as the former requires at least 1 character, as opposed to the latter which requires no characters